internal/storage/v2/grpc/trace_storage.proto

```diff
@@ -129,6 +129,14 @@ message FindTraceIDsResponse {
   repeated FoundTraceID trace_ids = 1;
 }
 
+// Chunking requirements:
```

Comment:
It's good to start definition with what it is - the type of payload in a stream that contains traces.